src/main/kotlin/me/lensvol/blackconnect/config/sections/ConnectionSection.kt

```diff
@@ -18,6 +18,7 @@ import javax.swing.JButton
 import javax.swing.JComponent
 import javax.swing.JPanel
 import javax.swing.JSpinner
+import javax.swing.JCheckBox
```

Comment:
sort

Reply:
inteliJ suggested to do 

> import javax.swing.*

im a bit afraid to switch to * imports. In python I'm not really doing it eagerly